lib/std/math.zig

```diff
@@ -1279,6 +1279,37 @@ test "lossyCast" {
     try testing.expect(lossyCast(u32, @as(f32, maxInt(u32))) == maxInt(u32));
 }
 
+/// Performs linear interpolation between *a* and *b* based on *t*.
+/// *t* must be between 0.0 and 1.0. T must be a float type.
+/// Passing `true` to `precise` makes results more accurate at the cost of speed.
+/// In the majority of cases using `false` is precise enough and is fastest.
+fn lerp(comptime T: type, precise: bool, a: T, b: T, t: T) T {
```

Comment:
Would prefer if `precise` were comptime, and perhaps was also instead an enum, e.g., `comptime strat: enum { precise, fast }`

Reply:
That sounds good.

Reply:
Only if the optimiser allows it to boil down as such, and I'd prefer not to have to rely on the optimiser if possible. If one really wants to make the decision at runtime, you can always make a branch like
```
_ = switch (runtime_strat) {
    inline else => |strat| lerp(f32, strat, a, b, t),
}
```

Reply:
Sure enough! That would've definitely been the right thing to do.

Reply:
I will know for next time.